docs/addons/nginx.md

```diff
@@ -115,8 +117,52 @@ After the above ingresses applied (ideally through a GitOps engine) you can now
 [http://riker.dev.my-domain.com](http://riker.dev.my-domain.com)
 [http://troi.dev.my-domain.com](http://troi.dev.my-domain.com)
 
+## TLS Termination and Certificates
+
+You can configure the NGINX add-on to terminate TLS at the load balancer and supply an ACM certificate through the platform blueprint.
+
+A certificate can be registered using a named [resource provider](../resource-providers/index.md).
+
+For convenience the framework provides a couple of common certificate providers:
+
+**Import Certificate**
+
+This case is used when certificate is already created and you just need to reference it with the blueprint stack:
+
+```typescript
+const myCertArn = "";
+ssp.EksBlueprint.builder()
+    .resourceProvider(GlobalResources.Certificate, new ImportCertificateProvider(myCertArn, "cert1-id"))
+    .addOns(new NginxAddOn({
+        certificateResourceName: GlobalResources.Certificate,
+        externalDnsHostname: 'my.domain.com'
+    }))
+    .teams(...)
+    .build(app, 'stack-with-cert-provider');
+```
+
+**Create Certificate**
+
+This case is used when certificate should be created with the blueprint stack. In this case, DNS validation needs a Route53 hosted zone to be provisioned (either along with the stack or separately) and registered as a resource provider.
```

Comment:
> This case is used... In this case, DNS validation ...

This sounds a bit repetitive. Maybe start the second sentence with, "Here, DNS validation needs ..."